README.md

```diff
@@ -7,6 +7,9 @@ SPDX-FileContributor: Leon Matthes <leon.matthes@kdab.com>
 SPDX-License-Identifier: MIT OR Apache-2.0
 -->
 
+<!-- markdownlint-disable MD034 -->
+<!-- ^^ MD034 complains about bare link in 'About KDAB' -->
```

Comment:
isn't this one correct ? shouldn't it be `<https://www.kdab.com>` instead of `https://www.kdab.com` ?

Reply:
I can change, I put this here because I think it's standard KDAB copypaste, but yes that's the right way.
There's another link somewhere in the repo that uses `[]()` but just has the same text in both. That should also be switched to `<>`